### ISSUE TITLE
Also i receive this warning

### ISSUE THREAD

**Issue by lelapin123** [PromtEngineer/localGPT#3]
I receive this error message in a new conda environnement:

C:\Users\(myname)\anaconda3\envs\privategpt\lib\site-packages\transformers\generation\utils.py:1255: UserWarning: You have modified the pretrained model configuration to control generation. This is a deprecated strategy to control generation and will be removed soon, in a future version. Please use a generation configuration file (see https://huggingface.co/docs/transformers/main_classes/text_generation)
  warnings.warn(

conda project is called privategpt but it is in fact fully dedicated to localgpt

**Comment by PromtEngineer** [comment]
I will have a look at it. Need to update the configuration.

**Comment by vaylonn** [comment]
same issue here

**Comment by ml2s** [comment]
same warning here on m1pro macbook

**Comment by Eleksar387** [comment]
Same here.

I am on Win10, 3060 12GB

**Comment by One-one-one-learner** [comment]
same warning on m1pro macbook

**Comment by pmorange** [comment]
I also wish to add that after the warning, nothing happens... I launch with CPU option so it may be that it's taking more time, but so much time that it seems it'll never give me an answer (I tried leaving it "thinking" for quite a long time already, and nothing is happening).

**Comment by DeDelner** [comment]
Same on a 16GB M2 Pro

**Comment by maotoledo** [comment]
Me too. I'm using an apple m1 pro

**Comment by barbarosalp** [comment]
Same here, Windows 11 Cuda 11.8 
```
conda install pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia
```

**Comment by duramaxlb7** [comment]
same on ubuntu 22.04

**Comment by yihchu** [comment]
same here, mac pro intel

**Comment by sesam123** [comment]
same here

**Comment by bidachon** [comment]
same on Mac book pro M2

**Comment by pmorange** [comment]
So the question remains: Is it working on any platform ? :-D

**Comment by duramaxlb7** [comment]
@pmorange The answer is yes, I have been able to get it to run after I received the above error message. Locally run on Ubuntu 22.04, GTX 1070.

I was experiencing a fail/exit each time the model loaded with a distinct pattern. Memory use by the GPU would reach 100%, then swap volume (originally 25GB) usage would increase to 100% (i.e. 25GB) and then fail.

I moved my partitions around and increased the swap volume to 125 GB and have successfully loaded the model each time after increasing the swap size. Swap volume usage seems to run around 33GB. Total Memory usage is 16.7GB (GTX 1070) + ~33GB swap volume.

Running directly in CLI, I was able to generate output using the pre-loaded constitution doc, 3x. times.

I'm not sure where my logs are being stored so couldn't figure out how long it was taking to generate the output. To resolve this, I loaded the model in VSC this morning (to better track timestamps, which I also can't seem to figure out).

This is the first time I have received the warning after increasing the swap size.
